src/components/CustomerAppeal/ReasonSelection/ReasonItem.tsx

```diff
@@ -40,7 +40,11 @@ export const ReasonItem: FunctionComponent<{
     >
       <View style={styles.reasonLayout}>
         <AppText>{c13nt(description)}</AppText>
-        <AppText style={styles.reasonAlert}>{descriptionAlert ?? ""}</AppText>
+        <AppText style={styles.reasonAlert}>
+          {descriptionAlert
+            ? c13nt(descriptionAlert, undefined, descriptionAlert)
```

Comment:
oooh actually I'm just curious, is there a reason we have both `c13nt` and `i18nt`? I know `i18nt` is for internalisation haha :D

Reply:
From what I understand, `c13nt` is for campaign-specific translations like how `pod` and `return-pod` campaigns have different success message from other campaigns. `i18nt`, on the other hand, is for generic translations that are on all campaigns, like `goToStatistics` or `OK` labels.

Reply:
Ahhh, got it got it. Thanks for the explanation!! \^^/